docker/package/Dockerfile-ubuntu

```diff
@@ -20,4 +20,4 @@ RUN opam install opam-bundle=0.4 --yes
 COPY docker/package/*.py /tezos-packaging/docker/package/
 COPY docker/package/defaults /tezos-packaging/docker/package/defaults
 COPY docker/package/scripts /tezos-packaging/docker/package/scripts
-ENTRYPOINT ["python3", "-m", "package.package_generator.py"]
+ENTRYPOINT ["python3", "-m", "package.package_generator"]
```

Comment:
Nice bugs.

Reply:
weird, that podman haven't caughted this error